popup.js

```diff
@@ -245,6 +245,7 @@ function sal4ReportFormat(report){
   new_report = {
       'serial': data.serial,
       'hostname': data.serial,
+      'device_id': data.google_device_identifier,
```

Comment:
I don't see where this is put into the new Sal 4 format - am I missing it somewhere? https://github.com/salopensource/sal/wiki/Checkin-and-checkin-modules